packages/meditrak-server/src/apiV2/dataElements/DeleteDataElements.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { DeleteHandler } from '../DeleteHandler';
+import {
+  assertAllPermissions,
+  assertAnyPermissions,
+  assertBESAdminAccess,
+  assertAdminPanelAccess,
+} from '../../permissions';
+import { assertDataElementPermissions } from './assertDataElementPermissions';
+
+export class DeleteDataElements extends DeleteHandler {
+  async assertUserHasAccess() {
+    // User has access to all permission groups for the data_element, plus tupaia admin panel
+    // Or is a BES admin
+    const dataElementPermissionChecker = accessPolicy =>
+      assertDataElementPermissions(accessPolicy, this.models, this.recordId);
+
+    await this.assertPermissions(
+      assertAnyPermissions([
```

Comment:
Probably out of scope for this PR but I suggest moving common permission checks to the base classes. 

A call with "BES Admin" always passes. Else, a call without "admin panel" always fails.

Reply:
There's a [ticket](https://linear.app/bes/issue/WAITP-597/obfuscate-permissions-error-messages) for cleaning up all these asserts, so in the meantime I'm leaving them consistent